mailpoet/mailpoet
# Emails can't be tell apart if the subject beginning is the sameEmail selector [MAILPOET-3618]

[MAILPOET-3618]

[MAILPOET-3618]: https://mailpoet.atlassian.net/browse/MAILPOET-3618?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

Testing instructions:
1. Go to WP Admin > MailPoet > Lists
2. Click to Add New Segment
3. Select `opened` segment type
4. Select any sent newsletter in the next dropdown
5. Click above MAILPOET-3618 link to enter Jira details of the pull request and verify you see properly improved design of the dropdown
PS. In case you don't see any sent newsletter in the second dropdown, feel free to send one so you'll see it next time.
